lldb/source/API/CMakeLists.txt

```diff
@@ -152,9 +152,7 @@ add_lldb_library(liblldb SHARED ${option_framework}
     lldbValueObject
     lldbVersion
     ${LLDB_ALL_PLUGINS}
-## BEGIN SWIFT
-    ${SWIFT_ALL_LIBS}
-## END SWIFT
+    swiftCore
```

Comment:
You probably need to do something like 
if(LLDB_ENABLE_SWIFT)
  SWIFT_ALL_LIBS="swiftCore"

Reply:
@JDevlieghere Can you review this, too?